### PR TITLE
Everybody is learning

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

**Comment by croaky** [comment]
I like this new `life` directory! I expect some good ones to go in there from my life coach @r00k.

File: life/everybody-is-learning.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Everybdy is learning
```
Comment (ajbrown) [inline life/everybody-is-learning.md]: onos, typos :-(

Comment (mehulkar) [inline life/everybody-is-learning.md]: ohnoes!

**Comment by mehulkar** [comment]
@ajbrown fixed!

**Comment by JoelQ** [comment]
I like it! :+1:

**Comment by JoelQ** [comment]
Merged in f3ce06cc74de7959ea88eba37572fdfcbb147519

@mehulkar Thanks for submitting this TIL! We'd like to credit you when we tweet about it. What is your twitter handle?

**Comment by mehulkar** [comment]
@JoelQ it's the same as github :smile: Thanks! [@mehulkar](//twitter.com/mehulkar)

**Pull request closed without merging**